src/store/ui.ts

```diff
@@ -468,4 +473,62 @@ export const uiStore = createPartialStore<UiStoreTypes>({
       dispatch("SET_PROGRESS", { progress: -1 });
     },
   },
+  SHOW_GENERATE_AND_SAVE_ALL_AUDIO_DIALOG: {
```

Comment:
とても細かいですが改行あったほうが良いかも
```suggestion

  SHOW_GENERATE_AND_SAVE_ALL_AUDIO_DIALOG: {
```